osc-server/src/main/java/org/osc/core/broker/service/securityinterface/BaseSecurityGroupInterfaceService.java

```diff
@@ -54,9 +51,9 @@ protected VirtualSystem validateAndLoad(EntityManager em, SecurityGroupInterface
             throw new VmidcBrokerValidationException("Security group interfaces cannot be created or updated for appliance manager that does not support policy mapping.");
         }
 
-        // Validate policies
-        Set<Policy> policies = PolicyEntityMgr.findPoliciesById(em, dto.getPolicyIds(),
-                vs.getDistributedAppliance().getApplianceManagerConnector());
+		// Validate policies
+		PolicyEntityMgr.findPoliciesById(em, dto.getPolicyIds(),
```

Comment:
Is this really validating something or should it check the result for null or empty? If cant investigate now place a TODO or create an issue to investigate later.

Reply:
We want to make sure the policy id's provided with API request are supported by manager. We look into the DB and validate the policies. If a policy/policies are not found we add it to the invalid policy list and provide the error message. Method name should be more detailed.